src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -1344,7 +1342,12 @@ def aks_browse(cmd, client, resource_group_name, name, disable_browser=False, li
     else:
         raise CLIError("Couldn't find the Kubernetes dashboard pod.")
     # launch kubectl port-forward locally to access the remote dashboard
-    logger.warning('Proxy running on %s', proxy_url)
+    if in_cloud_console():
+        response = requests.post('http://localhost:8888/openport/8001')
+        logger.warning('To view the console, please open {} in a new tab'.format(response.text))
```

Comment:
The response is a JSON, format like:
{
  "message": "Port 8001 is open",
  "url": "https://gateway06.westus.console.azure.com/n/cc-xxxxxx/cc-xxxxx/proxy/8001/"
}

you want to extract the "url" property here.

Reply:
done.